packages/tools/devtools/devtools-core/src/messaging/devtools-messages/ContainerList.ts

```diff
@@ -29,6 +29,11 @@ export namespace ContainerList {
 		 * List of keys for the Containers registered with the Devtools.
 		 */
 		containers: ContainerKey[];
+
```

Comment:
Nit: probably worth updating the docs on the outer `ContainerList` namespace to note that this includes both containers and data objects registered with the devtools.